.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 proptest-regressions
+
+.idea/
```

Comment:
Speaking as a fellow IntelliJ user, these sorts of editor-specific ignore directives are best kept in a local `excludesfile` so everyone doesn't have to add these to every repo they interact with

Reply:
This is me. @erik-3milabs feel free to remove it. I don't want to create a PR just for that.

Reply:
Done.